src/app/Fake.Azure.Kudu/Kudu.fs

```diff
@@ -17,7 +17,7 @@ let nextManifestPath = Environment.environVarOrDefault "NEXT_MANIFEST_PATH" Stri
 /// Used by KuduSync for tracking and diffing deployments.
 let previousManifestPath = Environment.environVarOrDefault "PREVIOUS_MANIFEST_PATH" String.Empty
 /// The path to the KuduSync application.
-let kuduPath = (Environment.environVarOrDefault "GO_WEB_CONFIG_TEMPLATE" ".") |> DirectoryInfo.ofPath
+let kuduPath = (Environment.environVarOrDefault "GO_WEB_CONFIG_TEMPLATE" ".") |> Path.GetDirectoryName 
```

Comment:
But this is incorrect if we fallback to the default of `.`, correct?

Reply:
```

Reply:
Fixed